networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,16 +13,65 @@ These release notes track the development of the Network Observability Operator
+
+[id="multi-arch-1.3"]
+==== Multiple architectures now supported 
+* Network Observability Operator can now run on an amd64, ppc64le, or arm64 architecture. Previously, it only ran on amd64. 
+
+[id="deprecated-features-1.3"]
+=== Deprecated features
+
+[id="authToken-host"]
+==== Deprecated configuration parameter setting
+The release of Network Observability Operator 1.3 deprecates the `spec.Loki.authToken` `HOST` setting. You must now only use the `FORWARD` setting.
+
+[id="network-observability-operator-1.3.0-bug-fixes"]
+=== Bug fixes
+* Previously, when the Operator was installed from the CLI, the `Role` and `RoleBinding` that are necessary for the Cluster Monitoring Operator to read the metrics were not installed as expected. The issue did not occur when the operator was installed from the web console. Now, either way of installing the Operator installs the required `Role` and `RoleBinding`. (link:https://issues.redhat.com/browse/NETOBSERV-1003[*NETOBSERV-1003*])
+
+* Since version 1.2, the Network Observability Operator can raise alerts when a problem occurs with the flows collection. Previously, due to a bug, the related configuration to disable alerts, `spec.processor.metrics.disableAlerts` was not working as expected and sometimes ineffectual. Now, this configuration is fixed so that it is possible to disable the alerts. (link:https://issues.redhat.com/browse/NETOBSERV-976[*NETOBSERV-976*])
+
+* Previously, when Network Observability was configured with `spec.loki.authToken` set to `DISABLED`, only a `kubeadmin` cluster administrator was able to view network flows. Other types of cluster administrators received authorization failure. Now, any cluster administrator is able to view network flows. (link:https://issues.redhat.com/browse/NETOBSERV-972[*NETOBSERV-972*])
+
+* Previously, a bug prevented users from setting `spec.consolePlugin.portNaming.enable` to `false`. Now, this setting can be set to `false` to disable port-to-service name translation. (link:https://issues.redhat.com/browse/NETOBSERV-971[*NETOBSERV-971*])
+
+* Previously, the metrics exposed by the console plugin were not collected by the Cluster Monitoring Operator (Prometheus), due to an incorrect configuration. Now the configuration has been fixed so that the console plugin metrics are correctly collected and accessible from the {product-title} web console. (link:https://issues.redhat.com/browse/NETOBSERV-765[*NETOBSERV-765*])
+
+//The below known issue is a WIP. Still trying to understand possible workarounds.
+[id="network-observability-operator-1.3.0-known-issues"]
+=== Known issues
+* When `processor.metrics.tls` is set to `PROVIDED` in the `FlowCollector`, the `flowlogs-pipeline` `servicemonitor` is not adapted to the TLS scheme. (link:https://issues.redhat.com/browse/NETOBSERV-1070[NETOBSERV-1070])
```

Comment:
[NETOBSERV-1070](https://issues.redhat.com/browse/NETOBSERV-1070) is marked Closed in 1.3 so should this issue be listed here?  If so, the description doesn't tell me what problem will occur.  I don't know what to make of:

`flowlogs-pipeline servicemonitor` is not adapted to the TLS scheme.

Reply:
@memodi @OlivierCazade Can you guys help me out with this Known Issue? I'm struggling to understand what context is missing that users will need to know to expect, aside from what is already stated. From our [slack](https://redhat-internal.slack.com/archives/C02939DP5L5/p1685460460174569) convo, it is my impression that there is no workaround.

Reply:
@skrthomas Yes, if user sets it to `PROVIDED` there is no workaround, there was a follow up created [NETOBSERV-1087](https://issues.redhat.com/browse/NETOBSERV-1087) to fix that, may be that should be listed as known issue?

Reply:
@memodi OK yeah, taking a look at this follow-up bug, _this_ seems like it is actually the known issue. I guess 1070 now has a patched fix but then 1087 is the remaining known issue?

Reply:
of 3 modes, DISABLED, AUTO and PROVIDED for tls scheme ,  1070 was filed for issues with AUTO and PROVIDED mode and 1070 fixed AUTO mode. PROVIDED mode will be fixed with 1087.

Reply:
will PROVIDED mode be fixed for 1.3? I'm guessing no, since I was asked to put this in the known issues.

Reply:
Correct,it won't be addressed for 1.3 timeframe, since @OlivierCazade had mentioned it involves crd changes.